subsys/disk/Kconfig

```diff
@@ -131,4 +131,23 @@ config DISK_SDHC_VOLUME_NAME
 	  Disk name as per file system naming guidelines.
 
 endif # DISK_ACCESS_SDHC
+
+config DISK_ACCESS_STM32_SDMMC
+	bool "STM32 SDMMC driver"
+	depends on HAS_STM32CUBE
+	select USE_STM32_HAL_SD
+	select USE_STM32_LL_SDMMC
```

Comment:
This should not be needed if you're using only the .h part of the ll api.
Can you check this is really needed (since I haven't seen use of ll .c function in the driver below) ?

Reply:
Without it I get link errors like:
```
modules/hal/stm32/stm32cube/stm32l4xx/drivers/src/stm32l4xx_hal_sd.c:947: undefined reference to `SDMMC_ConfigData'
```
Maybe USE_STM32_HAL_SD should depend on USE_STM32_LL_SDMMC?

Reply:
ok, let's keep it as is